### PR TITLE
fix rspec document

### PR DESCRIPTION
**Pull request by shibuiwilliam**
# What
- Fix rspec test example.

### CODE REVIEW

File: rspec/.gitignore [hunk 1]
```diff
@@ -42,3 +42,5 @@ yarn-debug.log*
 /reports/*
 subset.txt
 remainder.txt
+
+report/*
```
Comment (takanabe) [inline rspec/.gitignore:46]: Why don't use use either `report` or `reports`?

File: rspec/README.md [hunk 2]
```diff
@@ -1,24 +1,152 @@
-# README
+# rspec
```
Comment (takanabe) [inline rspec/README.md:1]:
```suggestion
# rspec

## Prerequisites 

- Install [rbenv](https://github.com/rbenv/rbenv)
- Run `rbenv install` to setup Ruby specified in `.ruby-version`
```

File: rspec/README.md [hunk 3]
```diff
@@ -1,24 +1,152 @@
...
-* Database initialization
+| Name   | Path   | HEAD Commit                              |
+|--------|--------|------------------------------------------|
+| ..     | ..     | 8b2e54601faa5e8b53c1bcad2527b77fb7ecd785 |
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+# Run test.
+$ bundle exec rspec --format RspecJunitFormatter --out report/rspec.xml
```
Comment (takanabe) [inline rspec/README.md:27]: ditto: user `reports` or `report`

**Review by takanabe — APPROVED** [review]
LGTM with nits

File: rspec/README.md (hunk 3 above)
Comment (shibuiwilliam) [reply rspec/README.md]: fixed
https://github.com/launchableinc/examples/pull/61/commits/cd5b46db4492480e1915715d98bd09114ce3ad4f

File: rspec/.gitignore (hunk 1 above)
Comment (shibuiwilliam) [reply rspec/.gitignore]: fixed
https://github.com/launchableinc/examples/pull/61/commits/cd5b46db4492480e1915715d98bd09114ce3ad4f

**Pull request merged**